introduction.md

```diff
@@ -4,4 +4,9 @@ This is part of the mastering Github course.
 
 h2. My favorite path
 
-JavaScript
\ No newline at end of file
+JavaScript
+
+h3. my deadly skills
+
+* Git
+* Everything else
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?